### DISCUSSION TITLE
window transparency

### DISCUSSION THREAD

**Discussion by apetl** [excalith/excalith-start-page#47] [Ideas]
A way to make the terminal window transparent/semi-transparent

**Comment by excalith** [comment]
Hey @apetl, 

I added both window opacity and window blur support on the latest release. Please have a look at [v.2.3.3 release notes](https://github.com/excalith/excalith-start-page/releases/tag/v2.3.3) and let me know what you think!

**Blur**
![a_blur](https://github.com/excalith/excalith-start-page/assets/9248355/9975d7a0-a3ad-44aa-bffd-ecec9bde94e0)

**Opacity**
![b_opacity](https://github.com/excalith/excalith-start-page/assets/9248355/b8519579-0a8f-43a3-bb1e-f3b7c0e58853)

**Reply by apetl** [reply]
Exactly what I wanted. Thanks
